dags/shared_tasks.py

```diff
@@ -170,7 +170,7 @@ def get_mapping_status_task(collection: dict, mapped_pages: list):
         status: success
         num_records_mapped: int
         page_exceptions: TODO
-        mapped_page_path: str, ex: 
+        mapped_page_path: str|None, ex:
```

Comment:
I get errors when I do this, but you can `from typing import Optional` and then `mapped_page_path: Optional[str]`